src/MessageBox/docs/AnnouncementExamples/PrimaryButtonNode.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { MessageBoxMarketerialLayout } from 'wix-style-react/MessageBox';
+import DropdownBase from 'wix-style-react/DropdownBase';
+import Button from 'wix-style-react/Button';
+import ChevronDown from 'wix-style-react/new-icons/ChevronDown';
```

Comment:
Replace with:
```suggestion
import ChevronDown from 'wix-ui-icons-common/ChevronDown';
```

I think it will solve this issue:
```
Module not found: Error: Can't resolve 'wix-style-react/new-icons/ChevronDown' in '/home/builduser/agent00/work/dbb751eb2b5c5b2e/src/MessageBox/docs/AnnouncementExamples'
```